lib/utils/context-utils.ts

```diff
@@ -8,5 +9,29 @@ import { Construct } from "@aws-cdk/core";
  * @returns 
  */
 export function valueFromContext(construct: Construct, key: string, defaultValue: any) {
-    return construct.node.tryGetContext(key) ?? defaultValue;
-}
\ No newline at end of file
+    return construct.node.tryGetContext(key) ?? defaultValue
+}
+
+export function kubernetesVersionContext(construct: Construct) {
```

Comment:
we can call it `kubernetesVersionFromContext` or we could use something like `k8sVersionFromContext` to reduce verbosity.